smdebug/tensorflow/tensor_ref.py

```diff
@@ -96,8 +99,14 @@ def from_variable(cls, variable, export_name=None, mode=None, original_tensor=No
                 # for mirrored variable value this will be the mirrored variable
                 original_tensor = variable
 
+            if is_tf_version_2x() and tf.executing_eagerly() and isinstance(variable, tf.Variable):
+                # In TF 2.X eager mode, TF throws an error if you try to access a tensor's name.
+                # We need to pass it in as a variable, not a tensor, to maintain the name.
+                tf_obj = variable
+            else:
+                tf_obj = variable.value()
```

Comment:
Is there a case where
v >=2.0 and eager execution but isinstance(variable, tf.Variable) ? 
Test case for all these cases.

Reply:
by default, this is the case. all test cases in tests/tensorflow2/test_keras hit this case. there is no tensor_Ref specific test case for tf 1.x or tf 2.x.

Reply:
edited the comment above, please check.